packages/jest-haste-map/package.json

```diff
@@ -11,22 +11,29 @@
   "types": "build/index.d.ts",
   "dependencies": {
     "@jest/types": "^24.6.0",
+    "anymatch": "^2.0.0",
     "fb-watchman": "^2.0.0",
     "graceful-fs": "^4.1.15",
     "invariant": "^2.2.4",
     "jest-serializer": "^24.4.0",
     "jest-util": "^24.6.0",
     "jest-worker": "^24.6.0",
     "micromatch": "^3.1.10",
-    "sane": "^4.0.3"
+    "sane": "^4.0.3",
+    "walker": "^1.0.7"
```

Comment:
Uses same dependency as already required by `sane`, just adding explicitly since we rely on it now too.